lib/presentation/widgets/molecules/main_counter_button_widget.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:water_tracker/custom_theme.dart';
+import 'package:water_tracker/generated/locale_keys.g.dart';
+import 'package:water_tracker/presentation/widgets/atoms/person_image_widget.dart';
+import 'package:water_tracker/presentation/widgets/molecules/assistant_widget.dart';
+
+class MainScreenCounterWidget extends StatelessWidget {
+  const MainScreenCounterWidget({super.key, required this.count, required this.maxCount, required this.cupWeight})
+      : counterCupWeight = count * cupWeight;
```

Comment:
rename pls. For me, it is unclear that counterCupWeight means max weight. I suggest maxWaterWeigth or waterLimit